arcgis_proxy/utils/geostore.py

```diff
@@ -3,23 +3,21 @@
 from arcgis_proxy.utils.reproject import reproject_esrijson
 
 
-def get_geostore(geostore_id, format='esri'):
-
+def get_geostore(geostore_id, api_key, format='esri'):
     """ make request to geostore microservice for user given geostore ID """
 
-    config = {
-            'uri': '/geostore/{}?format={}'.format(geostore_id, format),
-            'method': 'GET',
-    }
-
-    return request_to_microservice(config)
+    return request_to_microservice(
+        uri=f'/v2/geostore/{geostore_id}?format={format}',
```

Comment:
same comment I made on the other PR: adding /v2 may be an BC break, use /v1 instead